### ISSUE TITLE
6.1.2 Can't download/install OCR languages

### ISSUE THREAD

**Issue by jcurl** [cyanfish/naps2#42]
**Describe the bug**
After reinstalling 6.1.2 I want to add German to my OCR languages. I select it, it downloads three files, but after downloading it indicates an error.

**To Reproduce**
1. Go to OCR
2. Choose German
3. Click OK and follow the prompts.

**Expected behavior**
Downloading should work.

**Screenshots**
![image](https://user-images.githubusercontent.com/1563519/210230552-5486fad4-de60-4c1d-90b3-ffdae3ffa9d5.png)

**Desktop (please complete the following information):**
 - Windows 10 x64 19043

**Additional context**
It doesn't do this on all computers. I noticed an "errorlog.txt". Here's the output.

```text
2023-01-02 13:00:17.9832 Error preparing downloaded file System.ArgumentException: 1 is not a supported code page.
Parameter name: codepage
   at System.Text.Encoding.GetEncoding(Int32 codepage)
   at ICSharpCode.SharpZipLib.Zip.ZipFile.ReadEntries()
   at ICSharpCode.SharpZipLib.Zip.ZipFile..ctor(String name)
   at NAPS2.Dependencies.DownloadFormat.ZipDownloadFormat.Extract(String zipFilePath, String outDir)
   at NAPS2.Dependencies.DownloadFormat.ZipDownloadFormat.Prepare(String tempFilePath)
   at NAPS2.WinForms.FDownloadProgress.StartNextDownload()
2023-01-02 13:00:20.6056 Error preparing downloaded file System.ArgumentException: 1 is not a supported code page.
Parameter name: codepage
   at System.Text.Encoding.GetEncoding(Int32 codepage)
   at ICSharpCode.SharpZipLib.Zip.ZipFile.ReadEntries()
   at ICSharpCode.SharpZipLib.Zip.ZipFile..ctor(String name)
   at NAPS2.Dependencies.DownloadFormat.ZipDownloadFormat.Extract(String zipFilePath, String outDir)
   at NAPS2.Dependencies.DownloadFormat.ZipDownloadFormat.Prepare(String tempFilePath)
   at NAPS2.WinForms.FDownloadProgress.StartNextDownload()
2023-01-02 13:00:24.5930 Error preparing downloaded file System.ArgumentException: 1 is not a supported code page.
Parameter name: codepage
   at System.Text.Encoding.GetEncoding(Int32 codepage)
   at ICSharpCode.SharpZipLib.Zip.ZipFile.ReadEntries()
   at ICSharpCode.SharpZipLib.Zip.ZipFile..ctor(String name)
   at NAPS2.Dependencies.DownloadFormat.ZipDownloadFormat.Extract(String zipFilePath, String outDir)
   at NAPS2.Dependencies.DownloadFormat.ZipDownloadFormat.Prepare(String tempFilePath)
   at NAPS2.WinForms.FDownloadProgress.StartNextDownload()
2023-01-02 13:15:56.0298 Error preparing downloaded file System.ArgumentException: 1 is not a supported code page.
Parameter name: codepage
   at System.Text.Encoding.GetEncoding(Int32 codepage)
   at ICSharpCode.SharpZipLib.Zip.ZipFile.ReadEntries()
   at ICSharpCode.SharpZipLib.Zip.ZipFile..ctor(String name)
   at NAPS2.Dependencies.DownloadFormat.ZipDownloadFormat.Extract(String zipFilePath, String outDir)
   at NAPS2.Dependencies.DownloadFormat.ZipDownloadFormat.Prepare(String tempFilePath)
   at NAPS2.WinForms.FDownloadProgress.StartNextDownload()
2023-01-02 13:15:58.8717 Error preparing downloaded file System.ArgumentException: 1 is not a supported code page.
Parameter name: codepage
   at System.Text.Encoding.GetEncoding(Int32 codepage)
   at ICSharpCode.SharpZipLib.Zip.ZipFile.ReadEntries()
   at ICSharpCode.SharpZipLib.Zip.ZipFile..ctor(String name)
   at NAPS2.Dependencies.DownloadFormat.ZipDownloadFormat.Extract(String zipFilePath, String outDir)
   at NAPS2.Dependencies.DownloadFormat.ZipDownloadFormat.Prepare(String tempFilePath)
   at NAPS2.WinForms.FDownloadProgress.StartNextDownload()
```

Even after reinstalling (deinstalled first) and removing %APPDATA%\NAPS, download fails.
[errorlog.txt](https://github.com/cyanfish/naps2/files/10331515/errorlog.txt)

Of course, after downloading the components and installing them manually, I can now OCR in German as well as English.

**Comment by cyanfish** [comment]
This is a [known issue](https://sourceforge.net/p/naps2/tickets/941/) in 6.1.2, I believe it's related to using certain non-English languages for Windows.

But you shouldn't have to re-download after upgrading to 7.x and downgrading back to 6.x, that's an oversight on my part that I'll fix in the next beta.

**Comment by jcurl** [comment]
Confirmed this is fixed in 7.0b2. Closing.
